### PR TITLE
Add ktlint configuration to the project and fix code style

### PR DESCRIPTION
**Pull request by maxme**
Update travis to use lint and ktlint

### CODE REVIEW

**Comment by maxme** [comment]
@daniloercoli this one is ready for a second pass

File: README.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ compile ('com.github.wordpress-mobile.WordPress-Aztec-Android:aztec:develop-SNAP
 
 When Aztec is ready, we'll publish the artifact in Maven.
 
+## Using ktlint
+
+The FluxC project uses [ktlint](https://github.com/shyiko/ktlint) for Kotlin linting. You can run ktlint using `./gradlew ktlint`, and you can also run `./gradlew ktlintFormat` for auto-formatting. There is no IDEA plugin (like Checkstyle's) at this time.
```
Comment (daniloercoli) [inline README.md]: FluxC ? What is FluxC 😁

Comment (maxme) [reply README.md]: Argh, I can't even copy/paste text correctly ;(

**Comment by daniloercoli** [comment]
:shipit:

**Pull request merged**